rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -70,7 +69,7 @@ class CollectMatcherSpec extends FlatSpec with Matchers {
     result.par should be(
       inputs.par.prepend(
         EList(
-          List[Par](EVar(BoundVar(1)), EEvalBody(ChanVar(BoundVar(0))), GInt(7)),
+          List[Par](EVar(BoundVar(1)), EVar(BoundVar(0)), GInt(7)),
```

Comment:
I am really glad we are getting rid of this. I didn't like that `BoundVar`s could be existing both in a `Channel` and as a `Var`.